docs/overview/changelog.md

```diff
@@ -15,104 +15,232 @@ _Unreleased_
-  and "New direct message" buttons.
+  and "New direct message" buttons, and allowing topics to line-wrap
+  to 2 lines. The collapsed state of the direct messages section not
+  persists across reloads.
 - Redesigned the right sidebar buddy list visually, adding a new "THIS
   CONVERSATION" section showing just the users who've participated
   recently in the currently viewed conversation, and improving its
   filtering component.
+- Replaced the compact mode setting with flexible options for font
+  size and line spacing in the Zulip web app.
 - User groups are now much more powerful, supporting subgroups and
   with new permissions settings for who can administer the group, join
   it, or edit its membership. Permission for creating new groups is
   now separate from permission to administer all groups.
+- Invitation links can now encode initial groups for a user.
 - Most permissions settings in Zulip have been reimplemented with a
   new flexible system based on groups, and can have a value of any
   combination of roles, groups, and individual users. Groups can be
   nested as subgroups of other groups. Groups are now deactivated,
   rather than being fully deleted, so Zulip's audit logs still
   maintain a complete history of permissions changes.
+- Group settings was redesigned, with many new administration
+  permissions, and a new Permissions panel showing all the permissions
+  that membership in the group grants.
+- Channel administration permissions have been restructured to be
+  based on groups, with new settings for administrering channels,
+  adding subscribers, removing them, and being able to join channels.
+- The access control model for message content in channels has been
+  generalized to allow users to be able to join a private channel at
+  their convenience, without needing to be subscribed. Users who have
+  permission to subscribe themselves to a channel are able to preview
+  and search channel content, just like how non-guest users can access
+  public channels without first subscribing.
 - Added a right sidebar buddy list style option showing user avatars,
-  designed for smaller organizations.
+  designed for smaller organizations. You can now change the buddy
+  list style directly from a menu in the sidebar.
 - Changes the semantics for when messages are marked as read when
   scrolling the blue selection box past them and the bottom of the
   feed is not visible to avoid shifting one's place when briefly
   visiting a conversation.
 - Reworked the "archive channel" functionality to not mutate their
-  names or make content inaccessible, with a view towards supporting
-  unarchiving channels.
-- Significantly improved the performance of rendering messages in the
-  web application, and optimized common views to load near-instantly,
-  without making any network requests.
+  names or always make content inaccessible, with a view towards
+  supporting unarchiving channels.
```

Comment:
- Reworked the "archive channel" functionality to preserve channel names and content access, with a view...